fk/teamjoin.go

```diff
@@ -26,11 +26,16 @@ import (
 	fpr "github.com/fluidkeys/fluidkeys/fingerprint"
 	"github.com/fluidkeys/fluidkeys/out"
 	"github.com/fluidkeys/fluidkeys/pgpkey"
+	"github.com/fluidkeys/fluidkeys/team"
 	"github.com/fluidkeys/fluidkeys/ui"
 	"github.com/gofrs/uuid"
 )
 
 func teamJoin(teamUUID uuid.UUID) exitCode {
+	if exitCode := ensureNotAlreadyAMember(teamUUID); exitCode != 0 {
```

Comment:
same issue as #398 where this happens before selecting a key, so it's blocking requests across *all keys*